python/examples/integrations/writers/Writing_Reference_Profiles_to_WhyLabs.ipynb

```diff
@@ -0,0 +1,422 @@
+       "</div>"
+      ],
+      "text/plain": [
+       "  Transaction ID Customer ID  Quantity  Item Price  Total Tax  Total Amount  \\\n",
+       "0   T14259136777     C274477         1       148.9    15.6345      164.5345   \n",
+       "1    T7313351894     C267568         4        48.1    20.2020      212.6020   \n",
+       "2   T37745642681     C267098         1        10.9     1.1445       12.0445   \n",
+       "3   T13861409908     C271608         2       135.2    28.3920      298.7920   \n",
+       "4   T58956348529     C272484         4       144.3    60.6060      637.8060   \n",
+       "\n",
+       "       Store Type  Product Category Product Subcategory Gender  \\\n",
+       "0        TeleShop       Electronics     Audio and video      F   \n",
+       "1  Flagship store  Home and kitchen          Furnishing      M   \n",
+       "2  Flagship store          Footwear                Mens      F   \n",
+       "3             MBR          Footwear                Mens      F   \n",
+       "4        TeleShop          Clothing                Mens      F   \n",
+       "\n",
+       "  Transaction Type   Age  \n",
+       "0         Purchase  37.0  \n",
+       "1         Purchase  25.0  \n",
+       "2         Purchase  42.0  \n",
+       "3         Purchase  43.0  \n",
+       "4         Purchase  39.0  "
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "csv_url = \"https://whylabs-public.s3.us-west-2.amazonaws.com/datasets/tour/current.csv\"\n",
+    "df = pd.read_csv(csv_url)\n",
+    "\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 📊 Profiling the Data\n",
+    "\n",
+    "Let's profile the data with whylogs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import whylogs as why\n",
+    "from datetime import datetime, timezone\n",
+    "current_date = datetime.now(timezone.utc)\n",
+    "profile = why.log(df).profile()\n",
+    "profile.set_dataset_timestamp(current_date)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We're also setting the profile's dataset timestamp as the current datetime. If this is not set, the Writer would simply assign the current datetime automatically to the profile."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## ✍️ The WhyLabs Writer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, you can simply create a WhyLabsWriter object and use it to send your profiles.\n",
+    "The process is very similar to sending a regular profile. The only difference is you should the `option` method in order to pass `reference` as True and `reference_alias` as the alias for your Reference Profile:"
```

Comment:
Suggest following rephrasing, also probably good to use the same language in our platform e.g. "static" and while the log_reference api uses "alias" that term might not be helpful in this context or the new option parameter, maybe just call it something like reference profile name?
```suggestion
    "The process is very similar to sending a regular profile. The only difference is you should use the `option` method in order to specify `reference_profile_name` to indicate and name this profile as a static Reference Profile:"
```

Reply:
agreed, changed it.